### PR TITLE
Fixes VSTS Bug 860976: [Watson] System.NullReferenceException in

### PR DESCRIPTION
**Pull request by mkrueger**
MonoDevelop.Ide.dll!MonoDevelop.Ide.Gui.Document.GetContent
(MonoDevelop.Ide.dll)

https://devdiv.visualstudio.com/DevDiv/_workitems/edit/860976

### CODE REVIEW

**Comment by mkrueger** [comment]
GetContent can throw a null reference exception that shouldn't be but with the info from the bug I couldn't find the cause of the issue. 

But that should work-around it.

**Comment by slluis** [comment]
@mkrueger looking at the stack trace of the bug, it's clear it belongs to 8.0. I don't think it can happen in 8.1 or master.

File: main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor.Wrappers/TooltipProviderWrapper.cs [hunk 1]
```diff
@@ -56,11 +56,11 @@ public TooltipProviderWrapper (MonoDevelop.Ide.Editor.TooltipProvider provider)
...
 			foreach (var doc in IdeApp.Workbench.Documents) {
-				var textEditor = doc.Editor;
-				if (textEditor == null)
-					continue;
-				if (textEditor.FileName == editor.FileName)
-					return textEditor;
+				if (doc.FileName == editor.FileName) {
+					var textEditor = doc.Editor;
+					if (textEditor != null)
+    					return textEditor;
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.SourceEditor2/MonoDevelop.SourceEditor.Wrappers/TooltipProviderWrapper.cs]: Whitespace is mixed here.

**Comment by mkrueger** [comment]
@slluis ok then I close this PR.

**Pull request closed without merging**